.github/workflows/lint.yml

```diff
@@ -40,6 +40,9 @@ jobs:
 
     runs-on: ubuntu-20.04-self-hosted
 
+    container:
+      image: docker.io/tarantool/testing:ubuntu-focal
```

Comment:
What about `release-notes` and `checkpatch` jobs?

Reply:
fixed

Reply:
I updated docker images. In images, I fixed broken python and fix dependencies.